content/6_Advanced/SPneg.mdx

```diff
@@ -344,51 +344,72 @@ Timeline (USACO Camp):
 <CPPSection>
 
 ```cpp
-const int MOD = 1000000007;
+#include <iostream>
+#define ll long long
```

Comment:
1. `using ll = long long`
2. can you at least put `using std::cout`

Reply:
you don't want it  for cin.

Reply:
yeah bc even i still do std::cin